content/english/anti-vegan/humans-are-superior.md

```diff
@@ -0,0 +1,84 @@
+
+This creates what Warren refers to as an *oppressive conceptual framework*. An example is patriarchal thinking which falsely justifies the subordination of women by men based on nothing more than difference. Warren further claims that this logic of domination, and any conceptual framework which gives rise to it, "must be abolished  in order to make possible a meaningful notion of difference..."
+
+We need to move away from the oppressive conceptual framework that maintains *sameness* and only extends the moral community to those who are similar to us by application of some self-determined criteria. Warren refers to this as *arrogant perception*, resulting in a false notion of superiority which gives rise to the claim that those like us deserve moral consideration and others don't.
+
+An alternative *loving perception* "presupposes and maintains difference - a distinction between the self and other, between human and at least some nonhumans - in such a way that perception of the other as other *is* an expression of love for one who/which is recognised at the outset as independent, dissimilar, different."
+
+
+
+### A Thought Experiment
+
+Consider we discover an alien species (or more likely an intelligent species discovers us). That species has all of our abilities together with a technology far in advance of our own. They far exceed our mental capacity, can see a far greater colour spectrum and can hear sounds outside our aural range.  They can also communicate telepathically.
+
+If humans are superior to animals because of our unique capacity for complex problem solving then these aliens, with all our abilities  plus the unique ability of telepathy must be superior to us.
+
+- Does this mean the aliens are justified in harvesting the human species?
+
+If you believe that we can use animals *as we choose* because we deem ourselves superior (due to unique attributes) then you must accept that the aliens would be justified in using us *as they chose* for the same reason.
+
+If you believe the aliens would act wrongly in harvesting the human species then you need to reconsider your claim that human superiority gives us the right to use animals *as we choose*.
+
+### Superiority - so what?
+
+> ...to care for and about others is what being vegan is all about.
+
+Even assuming that humans' unique abilities do *as a matter of fact* denote superiority  it’s unclear why or how this is relevant. One of the cornerstones of morality is that we care for those less capable (or more vulnerable) than ourselves, and caring for others is what being vegan is all about.
+
+ 
```

Comment:
Can you add a `Citations` at the end with a bullet point list of the books and links referenced please? :)

Reply:
Yup.  Thought I already had. Bad Tery Bad :)